airflow/jobs/backfill_job_runner.py

```diff
@@ -183,7 +184,7 @@ def _update_counters(self, ti_status: _DagRunTaskStatus, session: Session) -> No
 
         filter_for_tis = TI.filter_for_tis(list(ti_status.running.values()))
         if filter_for_tis is not None:
-            refreshed_tis = session.query(TI).filter(filter_for_tis).all()
+            refreshed_tis = session.execute(select(TI).where(filter_for_tis)).scalars().all()
```

Comment:
```suggestion
            refreshed_tis = session.scalars(select(TI).where(filter_for_tis)).all()
```

Is this the same?

Reply:
Yes. Both are the same. I agree to a different PR to remove` all()`